packages/sync/src/conflicts/base/BaseStateLink.ts

```diff
@@ -1,31 +0,0 @@
-
-import { ApolloLink, NextLink, Operation } from "apollo-link";
```

Comment:
actually jf we can avoid removing that file. I have added some items here and want to avoid conflicts